doc/quality.rst

```diff
@@ -588,6 +774,45 @@ ExtBRC
+| ``-rc_init_occupancy $((2 * $bitrate))``              | patched        | This is initial buffer delay. You can vary this per your needs.          |
+|                                                       |                | Recommendation is to use 1/2 of bufsize.                                 |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| CBR                                                                                                                                               |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-b:v $bitrate -minrate $bitrate -maxrate $bitrate`` | patched        | This triggers CBR.                                                       |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-bufsize $((2 * $bitrate))``                        | patched        | You can vary bufsize per your needs. We recommend to avoid going below 1 |
+|                                                       |                | second to avoid quality loss. Buffer size of 2 seconds is recommended    |
+|                                                       |                | for VBR.                                                                 |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-rc_init_occupancy $bitrate``                       | patched        | This is initial buffer delay. You can vary this per your needs.          |
+|                                                       |                | Recommendation is to use 1/2 of bufsize.                                 |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| CBR & VBR common settings                                                                                                                         |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-b_strategy 1 -bf 7``                               | patched        | These 2 settings activate full 3 level B-Pyramid.                        |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-refs 3``                                           | patched        | 3 references is current Intel GPU maximum for AV1 encoder.               |
```

Comment:
Let's remove -refs 3 all together from AV1 command lines and this list.  Seems that this is ignored by VPL..  It sets this internally so it has no effect